test/tests/functional/pbs_sister_mom_crash.py

```diff
@@ -53,7 +53,9 @@ def test_sister_mom_crash(self):
             self.skipTest("test requires atleast two MoMs as input, " +
                           "use -p moms=<mom1:mom2>")
         sister_mom = self.moms.keys()[1]
-        pbsdsh_path = self.du.which(exe='pbsdsh')
+        conf = self.du.parse_pbs_config()
+        pbs_exec = conf['PBS_EXEC']
+        pbsdsh_path = os.path.join(pbs_exec, "bin", "pbsdsh")
```

Comment:
I don't think you need to parse pbs.conf.  I think self.server.pbs_conf['PBS_EXEC']